src/docs/community/migration.mdx

```diff
@@ -0,0 +1,1206 @@
+import { Divider } from '../../app/components';
+
+<!-- KEYWORDS: developer develop environment guide tutorial -->
+
+# Package Migration Instructions
+
+This guide will walk you through the process of migrating your project from the old `@pxblue`-scoped packages to the new `@brightlayer-ui`-scoped packages. We have maintained the version numbers when migrating, so in most cases you will simply need to change the package name and update the import statements throughout your application. For most packages, we have migrated the latest two major versions. If you are using an older version, you will need to upgrade before changing to the packages in the new scope.
+
+Peer dependencies have changed to **Brightlayer UI** scoped packages. Update your project's peer dependencies from `@pxblue`-scoped to new `@brightlayer-ui`-scoped packages.
+
+## @brightlayer-ui/angular-auth-workflow
+
+-   Change package namespace from `@pxblue` to `@brightlayer-ui`.
+-   Change all `<pxb->` component prefixes to `<blui->`.
+-   The below code snippets indicate the changes to be made for using Auth Module in your angular application.
+
+    ```sh
+    // package.json
+
+    @pxblue/angular-auth-workflow => @brightlayer-ui/angular-auth-workflow
+    ```
+
```

Comment:
Please test the formatting / styling on docit. You either use the three ticks code snippet format or the four-space indentation thing, but not both.

Reply:
Removed `sh`. Please review.

Reply:
I see what's happening... sorry.